src/similarity.py

```diff
@@ -9,17 +9,29 @@
 
 import utils
 
-
 SpectrumTuple = collections.namedtuple(
     "SpectrumTuple", ["precursor_mz", "precursor_charge", "mz", "intensity"]
 )
 
+#  return type
+SimilarityTuple = collections.namedtuple(
+    "SimilarityTuple",
+    [
+        "score",
+        "matched_intensity",
+        "max_contribution",
+        "n_greq_2p",
+        "matches"
+        # ,"matched_indices", "matched_indices_other"
```

Comment:
For plotting mirror spectra, we need to keep the matched peak indexes in the return value. Otherwise it's not possible to properly color the matching peaks.